mesonbuild/interpreter.py

```diff
@@ -2253,6 +2255,21 @@ def get_cross_property_method(self, args, kwargs) -> str:
                 return args[1]
             raise InterpreterException('Unknown cross property: %s.' % propname)
 
+    @noArgsFlattening
```

Comment:
Not needed since there is only one arg anyway. We now also have @typed_pos_args() to validate the number and  type of args. It's not used yet in interpreter but should be used for new API.

Reply:
Added `@types_pos_args` and removed manual type/num checking, thanks.